src/components/IFrameModal/IFrameModal.tsx

```diff
@@ -35,35 +36,33 @@ type IFrameModalProps = {
  */
 const IFrameModalMemo = (props: IFrameModalProps): JSX.Element => {
```

Comment:
Bitte noch eine Test-datei erstellen. Onclick ist gerade nicht in der coverage drin.